keyring/testing/backend_macOS.py

```diff
@@ -0,0 +1,173 @@
+# coding: utf-8
+
+"""
+Common test functionality for backends.
```

Comment:
This whole module appears to be a copy/paste of the normal tests. That's not good. Now bugs in one module will have to be fixed in the other. More importantly, it's unclear how this test differs from the normal tests. I could figure it out  by inspecting or running a diff on the two tests, but that's a lot of work.

It also means this backend may no longer be adequately tested compared to other backends.

Better is to have exactly one test for each aspect of behavior. In the case of this change, I'd expect to see a new test or class of tests that exercise a new user journey that uses the non-default keychain, probably in test_macOS.